### ISSUE TITLE
Window Toggle stopped working after newest Todoist update

### ISSUE THREAD

**Issue by rickstaa** [rickstaa/todoist-global-shortcuts-WIN10#5]
I just noticed that the Todoist window won't minimize anymore after the newest Todoist update. I will look into this when I have the time. In the meantime feel free to add relevant information to this topic.

**Comment by codedebugrepeat** [comment]
Hey Rick, nice job with your script! 🙌 
How do you invoke Todoist to minimize the window? 
Please add some details, if there's an issue we'll definitely want to look into it :)

**Comment by rickstaa** [comment]
Hey, @codedebugrepeat. Thanks a lot for subscribing to my workaround and monitoring the issue tracker. I just found some time in my schedule to look at the problem, and I fixed it. The problem appeared to be caused by me using the workaround, which was created for the regular Todoist app, with the "Todoist Preview" windows 10 app. I now added two separate installers in the AHK folder.

- Use the `installer.vbs` file to install the global shortcut for the regular **Todoist** windows 10 app.
- Use the `installer_todoist_preview.vbs` file to install the global shortcut for the **Todoist Preview** windows 10 app.

Can I ask you one question? Are you planning to add the global shortcuts as a native feature in the "Todoist" windows 10 app this year?
